python/cuml/dask/common/input_utils.py

```diff
@@ -100,7 +98,7 @@ def create(cls, data, client=None):
         multiple = isinstance(data, Sequence)
 
         datatype = 'cudf' if isinstance(first(data) if multiple else data,
```

Comment:
I think this rhs expression has finally grown up enough that it deserves to be a real multiline if